src/kagglehub/colab_cache_resolver.py

```diff
@@ -41,7 +41,10 @@ def __call__(self, h: ModelHandle, path: Optional[str] = None, *, force_download
         if force_download:
             logger.warning("Ignoring invalid input: force_download flag cannot be used in a Colab notebook")
 
-        logger.info(f"Attaching model '{h}' to your Colab notebook...")
+        if path:
+            logger.info(f"Attaching '{path}' from model '{h}' to your Colab notebook...")
+        else:
+            logger.info(f"Attaching model '{h}' to your Colab notebook...")
```

Comment:
I would also remove the following line below to make it consistent:

```
logger.info(f"Model '{h}' is attached.")
```

I also removed it for the Kaggle Cache Resolver to reduce the log spam: https://github.com/Kaggle/kagglehub/pull/40/files#diff-524b789095387583a1102b8801823f00e459138589c2cc02bbe6861705306db6L72

Reply:
Done.